x-pack/plugins/ml/public/application/services/new_job_capabilities/new_job_capabilities_service.ts

```diff
@@ -127,11 +81,6 @@ class NewJobCapsService {
     }
   }
 
-  public getFieldById(id: string): Field | null {
```

Comment:
`getAggById` could probably move the base class too

Reply:
Moved in 4e2f344ac831a47b06266517d1dbcd0dbd0f3c48